src/frontend/komponenter/Oppgaver/OppgaveList.tsx

```diff
@@ -28,223 +29,247 @@ interface IOppgaveListProps {
+    [ListFelt.AKTOER_ID, 'Bruker'],
+    [ListFelt.TILDELT_ENHETSNR, 'Enhet'],
+    [ListFelt.TILORDNET_RESSURS, 'Saksbehandler'],
+    [ListFelt.HANDLINGER, 'Handlinger'],
+    [ListFelt.BESKRIVELSE, 'Beskrivelse'],
+]);
+
+const oppgaveFeltMap = new Map<ListFelt, string>([
+    [ListFelt.OPPRETTET_TIDSPUNKT, 'opprettetTidspunkt'],
+    [ListFelt.OPPGAVETYPE, 'oppgavetype'],
+    [ListFelt.BEHANDLINGSTEMA, 'behandlingstema'],
+    [ListFelt.FRIST_FERDIGSTILLELSE, 'fristFerdigstillelse'],
+    [ListFelt.PRIORITET, 'prioritet'],
+    [ListFelt.AKTOER_ID, 'aktoerId'],
+    [ListFelt.TILDELT_ENHETSNR, 'tildeltEnhetsnr'],
+    [ListFelt.TILORDNET_RESSURS, 'tilordnetRessurs'],
+    [ListFelt.HANDLINGER, 'handlinger'],
+    [ListFelt.BESKRIVELSE, 'beskrivelse'],
+]);
+
+const ariaSortMap = new Map<FeltSortOrder, 'none' | 'descending' | 'ascending'>([
+    [FeltSortOrder.NONE, 'none'],
+    [FeltSortOrder.DESCENDANT, 'descending'],
+    [FeltSortOrder.ASCENDANT, 'ascending'],
+]);
+
+const sortLenkClassNameMap = new Map<
+    FeltSortOrder,
+    '' | 'tabell__th--sortert-desc' | 'tabell__th--sortert-asc'
+>([
+    [FeltSortOrder.NONE, ''],
+    [FeltSortOrder.DESCENDANT, 'tabell__th--sortert-desc'],
+    [FeltSortOrder.ASCENDANT, 'tabell__th--sortert-asc'],
+]);
+
+const initFeltOrder = new Map<ListFelt, FeltSortOrder>([
+    [ListFelt.OPPRETTET_TIDSPUNKT, FeltSortOrder.NONE],
+    [ListFelt.OPPGAVETYPE, FeltSortOrder.NONE],
+    [ListFelt.BEHANDLINGSTEMA, FeltSortOrder.NONE],
+    [ListFelt.FRIST_FERDIGSTILLELSE, FeltSortOrder.NONE],
+    [ListFelt.PRIORITET, FeltSortOrder.NONE],
+    [ListFelt.TILDELT_ENHETSNR, FeltSortOrder.NONE],
+    [ListFelt.TILORDNET_RESSURS, FeltSortOrder.NONE],
+]);
+
 const OppgaveList: React.FunctionComponent<IOppgaveListProps> = ({ innloggetSaksbehandler }) => {
-    const {
-        oppgaver,
-        sortOppgave,
-        hentOppgaveSide,
-        sideindeks,
-        forrigeSide,
-        nesteSide,
-        hentSidetall,
-    } = useOppgaver();
-
-    interface IFeltOrder {
-        [key: string]: boolean;
-    }
-
-    const initFeltOrder: IFeltOrder = {
-        opprettetTidspunkt: true,
-        oppgavetype: true,
-        behandlingstema: true,
-        fristFerdigstillelse: true,
-        prioritet: true,
-        tildeltEnhetsnr: true,
-        tilordnetRessurs: true,
-    };
+    const { oppgaver, sortOppgave, hentOppgaveSide } = useOppgaver();
 
-    const [feltOrder, settFeltOrder] = React.useState<IFeltOrder>(initFeltOrder);
+    const [feltSortOrder, settFeltSortOrder] = React.useState<Map<ListFelt, FeltSortOrder>>(
+        initFeltOrder
+    );
 
-    const onColumnSort = (felt: string) => {
-        sortOppgave(felt, feltOrder[felt]);
-        settFeltOrder({
-            ...feltOrder,
-            [felt]: !feltOrder[felt],
-        });
+    const onColumnSort = (felt: ListFelt) => {
+        sortOppgave(oppgaveFeltMap.get(felt)!, feltSortOrder.get(felt) !== FeltSortOrder.ASCENDANT);
```

Comment:
3 Forbidden non-null assertion here. Try to resolve the lint-issue with !

Reply:
Fixed👍